test/generateprecompiles.jl

```diff
@@ -0,0 +1,26 @@
+# To figure out what should be precompiled, run this script, then move
```

Comment:
Done. I thought Project.toml files were just for the top level of a project/package, or am I missing something?

Reply:
For someone without StatsPlots or RDataSets installed the test_examples 25 and 30 would still fail here. Furthermore, example 6 fails if FileIO is not installed.
Hence, I was thinking we could add a Project.toml with these dependencies, so people could just do `pkg> activate deps` before running the generateprecompiles.jl script.
However, this does not work anyway,  because in `@snoopc` a new julia process with the default project is started.
Maybe we should add a comment here that StatsPlots, RDataSets, FileIO and (obviously) SnoopCompile have to be installed to run this script.